src/XRP/default-xrp-client.ts

```diff
@@ -295,7 +296,10 @@ class DefaultXRPClient implements XRPClientDecorator {
       await this.getBalance(address)
       return true
     } catch (e) {
```

Comment:
Do we use `catch (e)` throughout `xpring-js`?

Single character variables seem like a bad practice to me. I personally use `err`, but I've seen `error` as well in `catch` blocks in our codebases.

But, if `catch(e)` is consistent with the rest of `xpring-js`, then maybe we can punt this to a future Find+Replace PR.

Reply:
+1 I'm sorry if I originally didn't comment. I'd favor `error` because I try to not use abbreviations in code.